packages/utils/src/print.ts

```diff
@@ -0,0 +1,14 @@
+export function PRINT_ERROR(msg: string) {
```

Comment:
Can we just move these into the chevrotain package?

Reply:
- There may be future utils that will be common/shared, so that gave me an incentive to keep the utils bundle.
- I believe the remaining three utils are all used in both the Parser and the Lexer, and I would like at some point in the future to extract the lexer to a separate package.